tokio-trace/src/subscriber.rs

```diff
@@ -15,5 +15,5 @@ pub fn with_default<T, S>(subscriber: S, f: impl FnOnce() -> T) -> T
 where
     S: Subscriber + Send + Sync + 'static,
 {
-    ::dispatcher::with_default(::Dispatch::new(subscriber), f)
+    ::dispatcher::with_default(&::Dispatch::new(subscriber), f)
```

Comment:
this is another case where it's not strictly necessary to bump the ref count. Not as serious as in `Span::enter` though, as this shouldn't happen nearly as often as entering a span.